models/repo.go

```diff
@@ -1646,7 +1646,7 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (repos []*Repository, _ int
 		}
 	}
 	if len(opts.Keyword) > 0 {
-		sess.And("repo.lower_name LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%")
+		sess.Where("repo.lower_name LIKE ? OR repo.description LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%", "%"+strings.ToLower(opts.Keyword)+"%")
```

Comment:
Hi @Unknwon While writing this small PR I looked into the source code of XORM for the `And` and `Where` methods to find that they end up resolving to the same `session.Statement.And` function:

```
// Where provides custom query condition.
func (session *Session) Where(query interface{}, args ...interface{}) *Session {
	session.Statement.Where(query, args...)
	return session
}

// And provides custom query condition.
func (session *Session) And(query interface{}, args ...interface{}) *Session {
	session.Statement.And(query, args...)
	return session
}
```

As you can see the XORM `And` method resolves to `session.Statement.And` and the `Where` method resolves to `session.Statement.Where`. The later looks like the below:

```
// Where add Where statement
func (statement *Statement) Where(query interface{}, args ...interface{}) *Statement {
	return statement.And(query, args...)
}
```

The `where` and `and` method appear to resolve to the same `session.Statement.And` function meaning they have the same functionality.

I have tested that this works in our production environment. You can see below a screen grab from our production environment showing a repository found by its description:

![image](https://cloud.githubusercontent.com/assets/464699/26820973/adb10332-4a9c-11e7-8062-e5cd97a6036d.png)


I believe that it will work the same if I replace the `where` for  `and`. Thinking about it, considering this is an SQL `AND( a OR b)` it may be more expressive of what is going on if I do replace it.

What do you think?